### PR TITLE
Walking on a tarp reveals it

### PR DESCRIPTION
**Pull request by psykzz**
Also passes damage onto the mob

### CODE REVIEW

File: code/game/objects/items/marine_gear.dm [hunk 1]
```diff
@@ -86,27 +86,37 @@
...
 	if(!opened && M)
-		M.bullet_act(Proj) //tarp isn't bullet proof; concealment, not cover; pass it on to the occupant.
+		M.bullet_act(proj) //tarp isn't bullet proof; concealment, not cover; pass it on to the occupant.
+
+/obj/structure/Crossed(mob/living/L)
+	if(!istype(L) || L.stat != CONSCIOUS)
+		return
+	// Walking on the tarp reveals it
+	open()
+	L.visible_message("<span class='notice'>You stepped on \the [src], revealing it!</span>", "<span class='notice'>[L] stepped on \the [src], revealing it!</span>")
```
Comment (jjpark-kb) [inline code/game/objects/items/marine_gear.dm:119]: was this meant to be /obj/structure ?

File: code/game/objects/items/marine_gear.dm [hunk 2]
```diff
@@ -86,27 +86,37 @@
...
 			qdel(src) //blown apart
 
-/obj/structure/closet/bodybag/tarp/bullet_act(obj/projectile/Proj)
+/obj/structure/closet/bodybag/tarp/bullet_act(obj/projectile/proj)
 	var/mob/M = locate() in src //need to be occupied
 	if(!opened && M)
-		M.bullet_act(Proj) //tarp isn't bullet proof; concealment, not cover; pass it on to the occupant.
+		M.bullet_act(proj) //tarp isn't bullet proof; concealment, not cover; pass it on to the occupant.
+
+/obj/structure/Crossed(mob/living/L)
```
Comment (jjpark-kb) [inline code/game/objects/items/marine_gear.dm:114]:
```suggestion
/obj/structure/closet/bodybag/tarp/Crossed(mob/living/L)
```

Comment (psykzz) [reply code/game/objects/items/marine_gear.dm]: It wasn't :D

**Comment by namjay1170** [comment]
care to buff it in someway to compensate its lack of usefulness?

**Pull request merged**